uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/drivers/roboteq_driver/src/CanopenInterface.cpp

```diff
@@ -97,8 +97,10 @@ CanopenInterface::CanopenInterface(canid_t roboteq_can_id, const std::string& if
   can_receive_filter[1].can_id = SDO_RESPONSE_COB_ID_OFFSET + roboteq::CanopenInterface::roboteq_can_id_;
   can_receive_filter[1].can_mask = (CAN_EFF_FLAG | CAN_RTR_FLAG | CAN_EFF_MASK);
 
-  //   struct timeval receive_timeout = {.tv_usec=500000}; //0.5 seconds
-  // setsockopt(socket_handle_, SOL_SOCKET, SO_RCVTIMEO, &receive_timeout, sizeof(receive_timeout));
+  constexpr unsigned RECEIVE_TIMEOUT_US{500000};  // 0.5 seconds
```

Comment:
why not chrono?

Reply:
This is a part of a struct used in a linux syscalls and those use plain old c